web-animations-2/Overview.bs

```diff
@@ -323,6 +323,33 @@ descendant effects and custom effects such that the first condition is:
 >     effect</a> or executing any <a>custom effects</a> associated with an
 >     <a>animation effect</a>.
 
+<h4 id='validating-a-css-numberish-time'>Validating a CSSNumberish time</h4>
+
+The procedure to <dfn>validate a CSSNumberish time</dfn> for an input
+value of <var>time</var> is based on the first condition that matches:
+
+    <div class="switch">
+
+    :    If <em>all</em> of the following conditions are true:
+         * The animation is associated with a [=progress-based timeline=], and
+         * <var>time</var> is not a CSSNumeric value with percent units:
+
+    ::    <a>throw</a> a <span class="exceptionname">TypeError</span>.
+    ::    return false;
+
+    :    If <em>all</em> of the following conditions are true:
+         * The animation is not associated with a [=progress-based timeline=],
+           and
```

Comment:
This is an incorrect indent. Bikeshed requires a 4-space (or tab) indent.

Reply:
Done